src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue

```diff
@@ -111,6 +111,11 @@ export default {
 			return !this.isOneToOne && (this.canFullModerate || this.mainConversation.participantType === PARTICIPANT.TYPE.GUEST_MODERATOR)
 		},
 
+		isOneToOne() {
```

Comment:
No wonder we haven't noticed it before :see_no_evil:

Reply:
It slipped from the radar